reconcile/utils/aws_api.py

```diff
@@ -1014,6 +1024,19 @@ def share_ami(
         }
         image.modify_attribute(LaunchPermission=launch_permission)
 
+    def get_cloudwatch_logs(self, account):
+        cloudwatch_logs = self._account_cloudwatch_client(account["name"])
+        log_groups = cloudwatch_logs.describe_log_groups()["logGroups"]
+        return log_groups
+
+    def set_cloudwatch_log_retention(self, account, log_groups, retention_days):
```

Comment:
set_cloudwatch_log_retention should only act on one log group instead of whole log_groups